styles/picrew/catppuccin.user.css

```diff
@@ -0,0 +1,1215 @@
+    @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; };
+  }
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+    color-scheme: if(@lookup = latte, light, dark);
+    body {
+      background-color: @base !important;
+      color: @text !important
+    }
+    /* language prompt */
+    .header-langbox {
+      background: @mantle !important;
+      color: @text !important;
+    }
+    .header-langbox-buttons button {
+      background: @subtext0 !important;
+      color: @base !important;
+    }
+    /* header */
+    .sitetop-Header {
+      background: @base !important;
+    }
+    .st-Header_Wrapper {
+      background: @base !important;
+    }
+    .st-Header_Wrapper .st-Header .sp-nav .nav-link li ul,
+    .st-Header_Wrapper .st-Header .sp-nav .nav-link li h2 {
+      border-top-color: @surface1 !important;
+    }
+    .st-Header_Wrapper .st-Header .st-Header_Logo {
+      filter: drop-shadow(0 0 1px @subtext0) !important;
+    }
+    .st-Header_Wrapper .st-Header .st-Header_Linkicon {
+      color: @subtext0 !important
+    }
+    .st-Header_Wrapper .st-Header .menu__line {
+      background-color: @subtext0 !important;
+    }
```

Comment:
```suggestion
    .st-Header_Wrapper {
      background: @yellow !important;
    }
    .st-Header_Wrapper .st-Header .sp-nav .nav-link li ul,
    .st-Header_Wrapper .st-Header .sp-nav .nav-link li h2 {
      border-top-color: @surface1 !important;
    }
    .st-Header_Wrapper .st-Header .st-Header_Logo {
        @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 62"><g fill="@{base}"><path d="M199.4 42.69a11.65 11.65 0 0 1-10.56 8.2 13.57 13.57 0 0 1-10.78-4.53c-5.24 6-15.42 6.21-19.82-.56-4.33 3.08-11 5.09-17.52 5.09-8.79-.29-13.1-3.22-19.06-7.93-6.34-5.9-15.36-1.2-19.53 6.12v10a2.4 2.4 0 0 1-4.79 0v-12.7a17.3 17.3 0 0 1-11.73 4.51 17 17 0 0 1-14.94-8.58 11.34 11.34 0 0 1-12.86.08c-4.86 9.09-15.06 14.3-28 14.3-10.19 0-18.66-3.2-24.18-9v11.38a2.4 2.4 0 0 1-4.8 0V40a2.44 2.44 0 0 1 1-2 2.4 2.4 0 0 1 3.45.8c3.8 8.5 12.67 13.08 24.49 13.13s22.42-5.22 24.85-14.77a2.37 2.37 0 0 1 1.23-1.37 2.41 2.41 0 0 1 3.12 1c2.06 3.73 8.37 3.72 10.43 0a2.41 2.41 0 0 1 4.08-.17 2.3 2.3 0 0 1 .33.75 12.33 12.33 0 0 0 23.61 0 2.4 2.4 0 0 1 4.66.05c.11.29.05 3.57.06 3.94 3.64-3.49 8.31-5.8 12.44-5.8 7.63-.26 10.24 4.68 15.57 7.65a21.2 21.2 0 0 0 10.58 2.93c7.11 0 13.7-2.58 17.07-6.21a2.42 2.42 0 0 1 3.49 1.34c2 6.51 11.46 6.55 14.55-.14a2.39 2.39 0 0 1 4.34-.26 8.62 8.62 0 0 0 8.38 5.27 6.9 6.9 0 0 0 6.29-4.87 2.39 2.39 0 0 1 4.56 1.46ZM64.17 46.1a2.39 2.39 0 0 0-2.39 2.4v10.57a2.4 2.4 0 1 0 4.79 0V48.5a2.39 2.39 0 0 0-2.4-2.4"/><path d="M18.91 21.4a.31.31 0 0 1 .32-.33 5 5 0 0 1 3.65 1.79A8.6 8.6 0 0 1 24.62 28a.29.29 0 0 1-.34.31c-3.92-.52-4.91-1.83-5.37-6.91m-1.22 0a.31.31 0 0 0-.32-.33 5 5 0 0 0-3.65 1.79A8.6 8.6 0 0 0 12 28a.29.29 0 0 0 .34.31c3.9-.52 4.89-1.83 5.35-6.91m-5.37 8.31A.3.3 0 0 0 12 30a8.62 8.62 0 0 0 1.74 5.18A5 5 0 0 0 17.37 37a.31.31 0 0 0 .32-.33c-.46-5.08-1.45-6.4-5.37-6.96m6.59 7a.31.31 0 0 0 .32.33 5 5 0 0 0 3.65-1.79A8.62 8.62 0 0 0 24.62 30a.3.3 0 0 0-.34-.32c-3.92.59-4.91 1.91-5.37 6.99Zm21.7-15.31a.31.31 0 0 0-.32-.33 5 5 0 0 0-3.65 1.79A8.6 8.6 0 0 0 34.9 28a.29.29 0 0 0 .34.31c3.92-.52 4.91-1.83 5.37-6.91m6.59 7a.29.29 0 0 0 .34-.31 8.6 8.6 0 0 0-1.74-5.18 5 5 0 0 0-3.65-1.79.31.31 0 0 0-.32.33c.46 5.03 1.45 6.34 5.37 6.9Zm-5.37 8.32a.31.31 0 0 0 .32.33 5 5 0 0 0 3.65-1.79A8.62 8.62 0 0 0 47.54 30a.3.3 0 0 0-.34-.32c-3.92.59-4.91 1.91-5.37 6.99Zm-6.59-7a.3.3 0 0 0-.34.32 8.62 8.62 0 0 0 1.74 5.18A5 5 0 0 0 40.29 37a.31.31 0 0 0 .32-.33c-.46-5.08-1.45-6.4-5.37-6.96Zm142.82-8a2.4 2.4 0 0 0-2.4 2.4v7.3a2.4 2.4 0 1 0 4.79 0v-7.3a2.39 2.39 0 0 0-2.39-2.44Zm-17.84-3.27a2.39 2.39 0 0 0-2.39 2.39v10.54a2.4 2.4 0 1 0 4.79 0V20.8a2.39 2.39 0 0 0-2.4-2.39Zm35.67 0a2.39 2.39 0 0 0-2.39 2.39v10.54a2.4 2.4 0 1 0 4.79 0V20.8a2.39 2.39 0 0 0-2.4-2.39Zm-42.72 13a2.41 2.41 0 0 1-2.4 2.4H128v.88a2.4 2.4 0 0 1-4.79 0 20 20 0 0 1 .41-3.77c2.18-12 16.69-16.37 25.39-8.54a12.4 12.4 0 0 1 4.16 8.96ZM129.25 29h18.66a8.2 8.2 0 0 0-2.16-3.12 10.87 10.87 0 0 0-7.39-2.67 10.11 10.11 0 0 0-9.11 5.79M85.61 18.41A17 17 0 0 0 70.67 27a11.17 11.17 0 0 0-6.49-2 11.3 11.3 0 0 0-5.51 1.41 35.6 35.6 0 0 0-6.58-15.85 5.7 5.7 0 0 1-.81-1.25c.11-7.95-9.56-11.65-15.19-6-.17.17-.32.34-.43.47a6 6 0 0 1-1.1-.06 42 42 0 0 0-4.8-.24 57 57 0 0 0-5.88.33C18.52-2.52 8.13 1 8.24 9.26a5.7 5.7 0 0 1-.81 1.25A36.36 36.36 0 0 0 .49 29.15a2.4 2.4 0 0 0 4.78.42 31.4 31.4 0 0 1 6-16.15A6.72 6.72 0 0 0 13 9.26a3.59 3.59 0 0 1 2.23-3.58 4.22 4.22 0 0 1 4.97 1.13c1.26 1.54 2.57 2 5.23 1.6a39 39 0 0 1 4.33-.21 39 39 0 0 1 4.33.21c2.8.41 4.08-.15 5.33-1.72a4.22 4.22 0 0 1 4.84-1 3.59 3.59 0 0 1 2.23 3.58 6.72 6.72 0 0 0 1.79 4.16 31.4 31.4 0 0 1 6 16.15c.17 1.64 0 3.07 1.45 3.9a2.4 2.4 0 0 0 3.31-1 6.25 6.25 0 0 1 10.38.09A2.4 2.4 0 0 0 73.8 32a12 12 0 0 1 11.81-8.8A12.26 12.26 0 0 1 96 28.65a2.4 2.4 0 1 0 4-2.65 17 17 0 0 0-14.39-7.59"/></g></svg>');
        background-image: url("data:image/svg+xml;charset=utf8,@{svg}");
        filter: none;
    }
    .st-Header_Wrapper .st-Header .st-Header_Linkicon {
      color: @base !important
    }
    .st-Header_Wrapper .st-Header .menu__line {
      background-color: @base !important;
    }
```

Reply:
You're a lifesaver for theming the logo svg, I had no idea how to do that 😭 I'm committing this ASAP

Reply:
Lol happy to help! We've recently created a tips and tricks document that covers this, see https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md.